src/components/App.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import Characters from './characters/Characters';
```

Comment:
```suggestion
import { Characters } from './characters/Characters';
```
There are multiple exports coming from Characters.js, so the syntax above is needed to compile without warnings in the browser.